### ISSUE TITLE
[Bug] unit conversion is missing for TEMPO L2 V03 troposphric NO2 ObsError

### ISSUE THREAD

**Issue by HyundeokChoi-NOAA** [JCSDA-internal/ioda-converters#1559]
## Current behavior (describe the bug)
>The converter  doesn't convert the "tropospheric NO2 ObsError" unit from [molecules/cm2] to [mole/m2] for TEMPO level 2 version 03 product.

## To Reproduce

> What computer are you running on?
HERA
> Steps to reproduce the behavior
Test with TEMPO level 2 version 03 product for tropospheric NO2. 
## Additional information (optional)
                # error calculation:
                err = ncd.groups['product'].variables[err_name+'_uncertainty'][:].ravel()
                if self.v3:
                    if self.columnType == "total" or self.columnType == "stratosphere":
                        sys.exit("no error with total and strato NRT product")
                else:
                    err = err * conv * col_amf / tot_amf

**Comment by HyundeokChoi-NOAA** [comment]
## Revised as below
                # error calculation:
                err = ncd.groups['product'].variables[err_name+'_uncertainty'][:].ravel()
                if self.v3:
                    if self.columnType == "total" or self.columnType == "stratosphere":
                        sys.exit("no error with total and strato NRT product")
                    else:
                       err = err * conv * col_amf / tot_amf
                else:
                    err = err * conv * col_amf / tot_amf

**Comment by jeromebarre** [comment]
I remember correcting this before the TEMPO meeting. I think I forgot to PR my branch. Let me do this ASAP.

**Comment by HyundeokChoi-NOAA** [comment]
I saw you revised the "cld_fra", too. It was too strict. 
Thank you!!

**Comment by jeromebarre** [comment]
No problems. For the cloud fraction 0.5 might then be a bit too loose. There is no real consensus as far as I can tell on the correct value. It really depends on your DA results after all. I suggest using something loose to remove the pixels that we know for sure will be bad at first and then use a QC filter to adjust during the DA.

**Comment by HyundeokChoi-NOAA** [comment]
I agree. Also, I would like to add "solar_zeneth_angle" to the MetaData which is recommended by TEMPO science team (< 70)  so that we can use it as a filter during the DA.
